packages/client/src/pages/Leaderboard/Leaderboard.tsx

```diff
@@ -32,56 +33,64 @@ const Leaderboard: FC = () => {
   })
   const getHeaderButton = (text: string, field: string) => (
     <Button
-      variant="ghost"
-      colorScheme={ratingFieldName === field ? 'blue' : 'whiteAlpha'}
+      variant='unstyled'
+      background={'none'}
+      color={ratingFieldName === field ? `${themeColors.INVERTED_TEXT}` : `${themeColors.INVERTED_TEXT}`}
       pl={0}
       onClick={() => setRatingFieldName(field)}
     >
       {text}
-      {ratingFieldName === field && (
-        <TriangleDownIcon color='blue' />
-      )}
+      {ratingFieldName === field 
+        ? <TriangleDownIcon color={`${themeColors.INVERTED_TEXT}`} />
+        : <TriangleDownIcon color={`${themeColors.INVERTED_BACKGROUND}`} />
+      }
```

Comment:
[Предложение]
Вот так выглядит попроще, компонент же один и тот же, нет смысла именно его в тернарник добавлять 

```suggestion
      {<TriangleDownIcon color={ratingFieldName === field ? `${themeColors.INVERTED_TEXT}` : `${themeColors.INVERTED_BACKGROUND}`} />}
```

Reply:
[Вопрос]
А у переменных тип не строка разве? Их точно нужно оборачивать в строку?